bigbluebutton-html5/private/static/guest-wait/guest-wait.html

```diff
@@ -298,8 +299,11 @@
             }
 
             if (status === 'ALLOW') {
-              disableAnimation();
-              window.location = data.response.url;
+              updateLobbyMessage(_('app.guest.allow'));
+              setTimeout(() => {
+                disableAnimation();
+                window.location = data.response.url;
+              }, MESSAGE_TIMEOUT);
```

Comment:
Could you please add an inline comment on why you require a timeout? (I am assuming there's accessibility limitation on having to display a message for minimum of 3 sec, ... right?)

Reply:
Yeah 3 seconds was the minimum, I'll update with the comment.